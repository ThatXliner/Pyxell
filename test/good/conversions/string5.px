
print 42.42, -"987.987654321098".toFloat()
print '~' + 1234567890123.45194183604523642.toString()

print -"-.000000000000000000000000000000001".toFloat(),
      "0{+6700000000000000000000000000000000000000000.0}".toFloat()
