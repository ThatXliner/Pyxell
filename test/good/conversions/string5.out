42.42 -987.987654321098
~1234567890123.45
1e-33 6.7e+42
